api/repositories/service_binding_repository.go

```diff
@@ -69,6 +76,10 @@ type ServiceBindingRecord struct {
 	Ready               bool
 }
 
+type ServiceBindingParametersRecord struct {
```

Comment:
I would say that this type is redundant. We can just make the `GetServiceBindingParameters` method return `map[sting]any`. This is similar to the presenter case above, as the struct does not really add anything.